docs/_specification/1.2-DRAFT/appendix/implementation-notes.md

```diff
@@ -136,14 +144,14 @@ measures, e.g.
 `gpg --detach-sign --armor --output tagmanifest-sha512.txt.asc tagmanifest-sha512.txt`
 in combination with a secure PGP key exchange or equivalent trust network.
 
-#### Base URI in BagIt
+##### Base URI in BagIt
 
 The arcp specification suggests how [BagIt UUID identifiers] can be used to calculate the base URI of a bag, see section [Establishing a base URI inside a ZIP file](relative-uris#establishing-a-base-uri-inside-a-zip-file).  For this purpose it is RECOMMENDED that `bag-info.txt` includes a fresh UUID like:
 
     External-Identifier: urn:uuid:24e51ca2-5067-4598-935a-dac4e327d05a
 
 
-#### Referencing external files
+##### Referencing external files
```

Comment:
further down this file, the headings "Snapshots of external files" and "Example of wrapping a BagIt bag in an RO-Crate" also need an extra `#` in their heading level